packages/cli/src/daemon/Daemon.definition.ts

```diff
@@ -21,7 +22,8 @@ const definition: ICommandDefinition = {
                  "Daemon-mode runs the CLI command processor as a daemon to improve performance.",
     children: [
         EnableCommand,
-        DisableCommand
+        DisableCommand,
+		RestartCommand
```

Comment:
FYI - You can detect lint warnings, while you are still in your local working copy of your branch with the 'npm run lint' command. I pasted an example below:

```
npm run lint

> lint
> eslint "packages/**/*.ts" "**/__tests__/**/*.ts"


C:\ourstuff\repos\zowe-cli\packages\cli\src\daemon\DaemonDecider.ts
  34:1  warning  Expected indentation of 4 spaces but found 1 tab  @typescript-eslint/indent

✖ 1 problem (0 errors, 1 warning)
  0 errors and 1 warning potentially fixable with the `--fix` option.
```